core/templates/dev/head/css/oppia.css

```diff
@@ -1075,7 +1075,43 @@ textarea {
   right: 0;
 }
 
-/* Rules for the tiles in the "My Explorations" page. */
+.oppia-dashboard-tabs {
+  border-bottom: 1px solid #eee;
+  display: flex;
```

Comment:
Yes, for now. It's already used elsewhere, so if we want to remove it we'll have to do it in a number of places as a separate issue.